### DISCUSSION TITLE
Comboboxglyth styling

### DISCUSSION THREAD

**Discussion by WolframCrem** [AvaloniaUI/Avalonia#14635] [General]
Hello Everyone,
how can I change / remove the two black bars on the top and the bottom when the combobox is opened
![image](https://github.com/AvaloniaUI/Avalonia/assets/39000530/97959d7c-fec3-4f97-9e50-60987d6e8904)
there is a lighter gray thingy that i want to be removed if possible

thanks

**Comment by gehongyan** [comment]
See the `ComboBox` source code:

https://github.com/AvaloniaUI/Avalonia/blob/a33b39e8e021e01290d8d714f51ae17752fb03fd/src/Avalonia.Themes.Fluent/Controls/ComboBox.xaml#L123-L123

The mentioned gray bars result from the `Background` of the `Border` inside the `Popup`. The value is provided by `ComboBoxDropDownBackground` dynamic resource, hence the available modification:

```xaml
<ComboBox Width="160" IsDropDownOpen="True">
    <ComboBox.Resources>
        <ResourceDictionary>
            <SolidColorBrush x:Key="ComboBoxDropDownBackground" Color="Black" />
        </ResourceDictionary>
    </ComboBox.Resources>
    <ComboBoxItem Background="Black">Power Saving</ComboBoxItem>
    <ComboBoxItem Background="Black">Balanced</ComboBoxItem>
    <ComboBoxItem Background="Black">Max Performances</ComboBoxItem>
</ComboBox>
```

![image](https://github.com/AvaloniaUI/Avalonia/assets/21241496/ee1c444c-993b-4f36-bae3-8fd18bb8a048)

**Comment by WolframCrem** [comment]
That solved it, thank you!
